{"uid":"8ad47e310e614550","name":"test_send_message","fullName":"tests.test_example.ExampleTestCase#test_send_message","historyId":"3bc4f3765c9b31fcf733254dd25c049d","time":{"start":1757403050290,"stop":1757403052449,"duration":2159},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//3.254.161.142%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%203.254.161.142%3A3000.\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nUnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\ncheckReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nonNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nemit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25","statusTrace":"self = <test_example.ExampleTestCase testMethod=test_send_message>\n\n    def test_send_message(self):\n>       self.home.open(OLLAMA_URL)\n\ntests/test_example.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/home_page.py:21: in open\n    self.driver.get(url)\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:483: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:458: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fd7b5556610>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"Reached error page: about:neterror?e=connection....sys.mjs:156:19\\\\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25\\\\n\"}}'}\n\n    def check_response(self, response: dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                try:\n                    value = json.loads(value_json)\n                    if isinstance(value, dict):\n                        if len(value) == 1:\n                            value = value[\"value\"]\n                        status = value.get(\"error\", None)\n                        if not status:\n                            status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                            message = value.get(\"value\") or value.get(\"message\")\n                            if not isinstance(message, str):\n                                value = message\n                                message = message.get(\"message\")\n                        else:\n                            message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//3.254.161.142%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%203.254.161.142%3A3000.\nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nE       UnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\nE       checkReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nE       onNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nE       emit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nE       receiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25\n\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py:232: WebDriverException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":5,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_ExampleTestCase","time":{"start":1757403044466,"stop":1757403044466,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"afterStages":[{"name":"_unittest_setUpClass_fixture_ExampleTestCase::0","time":{"start":1757403052473,"stop":1757403052473,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_example"},{"name":"subSuite","value":"ExampleTestCase"},{"name":"host","value":"runnervmf4ws1"},{"name":"thread","value":"2691-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_example"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"d5b1a49c710858b8","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//3.254.161.142%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%203.254.161.142%3A3000.\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nUnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\ncheckReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nonNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nemit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25","time":{"start":1757403042266,"stop":1757403044790,"duration":2524}},{"uid":"8eae44bcf05859d5","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: Reached error page: about:neterror?e=connectionFailure&u=http%3A//3.254.161.142%3A3000/&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%203.254.161.142%3A3000.\nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:199:5\nUnknownError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:910:5\ncheckReadyState@chrome://remote/content/marionette/navigate.sys.mjs:59:24\nonNavigation@chrome://remote/content/marionette/navigate.sys.mjs:348:39\nemit@resource://gre/modules/EventEmitter.sys.mjs:156:19\nreceiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.sys.mjs:33:25","time":{"start":1757403040274,"stop":1757403042997,"duration":2723}},{"uid":"7021b93ff029202e","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=139.0.7258.154)\nStacktrace:\n#0 0x55e778c3108a <unknown>\n#1 0x55e7786d0a70 <unknown>\n#2 0x55e7786c7cf1 <unknown>\n#3 0x55e7786b9099 <unknown>\n#4 0x55e7786bade0 <unknown>\n#5 0x55e7786b943e <unknown>\n#6 0x55e7786b8daa <unknown>\n#7 0x55e7786b8a85 <unknown>\n#8 0x55e7786b68ee <unknown>\n#9 0x55e7786b70ca <unknown>\n#10 0x55e7786d3f99 <unknown>\n#11 0x55e77876ed65 <unknown>\n#12 0x55e7787483e2 <unknown>\n#13 0x55e77876e14f <unknown>\n#14 0x55e7787481b3 <unknown>\n#15 0x55e77871459b <unknown>\n#16 0x55e778715971 <unknown>\n#17 0x55e778bf625b <unknown>\n#18 0x55e778bf9fa9 <unknown>\n#19 0x55e778bdd339 <unknown>\n#20 0x55e778bfab58 <unknown>\n#21 0x55e778bc1c1f <unknown>\n#22 0x55e778c1e118 <unknown>\n#23 0x55e778c1e2f6 <unknown>\n#24 0x55e778c30066 <unknown>\n#25 0x7fefd869caa4 <unknown>\n#26 0x7fefd8729c3c <unknown>","time":{"start":1757403037860,"stop":1757403038626,"duration":766}},{"uid":"e214f881dcdf0226","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=139.0.7258.154)\nStacktrace:\n#0 0x5574feb4b08a <unknown>\n#1 0x5574fe5eaa70 <unknown>\n#2 0x5574fe5e1cf1 <unknown>\n#3 0x5574fe5d3099 <unknown>\n#4 0x5574fe5d4de0 <unknown>\n#5 0x5574fe5d343e <unknown>\n#6 0x5574fe5d2daa <unknown>\n#7 0x5574fe5d2a85 <unknown>\n#8 0x5574fe5d08ee <unknown>\n#9 0x5574fe5d10ca <unknown>\n#10 0x5574fe5edf99 <unknown>\n#11 0x5574fe688d65 <unknown>\n#12 0x5574fe6623e2 <unknown>\n#13 0x5574fe68814f <unknown>\n#14 0x5574fe6621b3 <unknown>\n#15 0x5574fe62e59b <unknown>\n#16 0x5574fe62f971 <unknown>\n#17 0x5574feb1025b <unknown>\n#18 0x5574feb13fa9 <unknown>\n#19 0x5574feaf7339 <unknown>\n#20 0x5574feb14b58 <unknown>\n#21 0x5574feadbc1f <unknown>\n#22 0x5574feb38118 <unknown>\n#23 0x5574feb382f6 <unknown>\n#24 0x5574feb4a066 <unknown>\n#25 0x7efcc929caa4 <unknown>\n#26 0x7efcc9329c3c <unknown>","time":{"start":1757403037495,"stop":1757403038119,"duration":624}},{"uid":"14025a99f3aed5ad","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=139.0.7258.154)\nStacktrace:\n#0 0x5637d3e0308a <unknown>\n#1 0x5637d38a2a70 <unknown>\n#2 0x5637d3899cf1 <unknown>\n#3 0x5637d388b099 <unknown>\n#4 0x5637d388cde0 <unknown>\n#5 0x5637d388b43e <unknown>\n#6 0x5637d388adaa <unknown>\n#7 0x5637d388aa85 <unknown>\n#8 0x5637d38888ee <unknown>\n#9 0x5637d38890ca <unknown>\n#10 0x5637d38a5f99 <unknown>\n#11 0x5637d3940d65 <unknown>\n#12 0x5637d391a3e2 <unknown>\n#13 0x5637d394014f <unknown>\n#14 0x5637d391a1b3 <unknown>\n#15 0x5637d38e659b <unknown>\n#16 0x5637d38e7971 <unknown>\n#17 0x5637d3dc825b <unknown>\n#18 0x5637d3dcbfa9 <unknown>\n#19 0x5637d3daf339 <unknown>\n#20 0x5637d3dccb58 <unknown>\n#21 0x5637d3d93c1f <unknown>\n#22 0x5637d3df0118 <unknown>\n#23 0x5637d3df02f6 <unknown>\n#24 0x5637d3e02066 <unknown>\n#25 0x7fab09c9caa4 <unknown>\n#26 0x7fab09d29c3c <unknown>","time":{"start":1757403036029,"stop":1757403036753,"duration":724}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"8ad47e310e614550.json","parameterValues":[]}